// @contactlab/ds-tokens/constants/colors.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.fluoAccent900 = exports.fluoAccent600 = exports.fluoAccent300 = exports.fluoBase900 = exports.fluoBase600 = exports.fluoBase300 = exports.black = exports.white = exports.error = exports.warning = exports.success = exports.info = exports.base900 = exports.base800 = exports.base700 = exports.base600 = exports.base500 = exports.base400 = exports.base300 = exports.base200 = exports.base100 = exports.base = exports.accent900 = exports.accent800 = exports.accent700 = exports.accent600 = exports.accent500 = exports.accent400 = exports.accent300 = exports.accent200 = exports.accent100 = exports.accent = void 0;
/**
 * The Contactlab Marketing Cloud color palette.
 */
exports.accent = '#1890ff';
exports.accent100 = '#e6f7ff';
exports.accent200 = '#bae7ff';
exports.accent300 = '#91d5ff';
exports.accent400 = '#69c0ff';
exports.accent500 = '#40a9ff';
exports.accent600 = '#1890ff';
exports.accent700 = '#096dd9';
exports.accent800 = '#0050b3';
exports.accent900 = '#003a8c';
exports.base = '#141414';
exports.base100 = '#fafafa';
exports.base200 = '#f5f5f5';
exports.base300 = '#f0f0f0';
exports.base400 = '#d9d9d9';
exports.base500 = '#bfbfbf';
exports.base600 = '#8c8c8c';
exports.base700 = '#595959';
exports.base800 = '#434343';
exports.base900 = '#141414';
exports.info = '#1890ff';
exports.success = '#52c41a';
exports.warning = '#faad14';
exports.error = '#f5222d';
exports.white = '#fff';
exports.black = '#000';
exports.fluoBase300 = '#e9e8ff';
exports.fluoBase600 = '#b4afff';
exports.fluoBase900 = '#7168ff';
exports.fluoAccent300 = '#e1f9ef';
exports.fluoAccent600 = '#9bebd0';
exports.fluoAccent900 = '#00c9bd';
