// @contactlab/ds-tokens/constants/custom-media.js
"use strict";
/**
 * The Contactlab Marketing Cloud custom media queries
 */
Object.defineProperty(exports, "__esModule", { value: true });
exports.highDensity = exports.finePointer = exports.darkScheme = exports.lightScheme = exports.reducedMotion = exports.portrait = exports.landscape = exports.toXLarge = exports.toLarge = exports.toMedium = exports.toSmall = exports.toXSmall = exports.fromXLarge = exports.fromLarge = exports.fromMedium = exports.fromSmall = exports.fromXSmall = void 0;
// --- Mobile first
exports.fromXSmall = 'min-width: 36em'; // 576px
exports.fromSmall = 'min-width: 48em'; // 768px
exports.fromMedium = 'min-width: 62em'; // 992px
exports.fromLarge = 'min-width: 75em'; // 1200px
exports.fromXLarge = 'min-width: 100em'; // 1600px
// --- Desktop first
exports.toXSmall = 'max-width: 36em'; // 576px
exports.toSmall = 'max-width: 48em'; // 768px
exports.toMedium = 'max-width: 62em'; // 992px
exports.toLarge = 'max-width: 75em'; // 1200px
exports.toXLarge = 'max-width: 100em'; // 1600px
// --- Orientation
exports.landscape = 'orientation: landscape';
exports.portrait = 'orientation: portrait';
// --- a11y
exports.reducedMotion = 'prefers-reduced-motion: reduce';
exports.lightScheme = 'prefers-color-scheme: light';
exports.darkScheme = 'prefers-color-scheme: dark';
// --- Misc
exports.finePointer = '(hover: hover) and (pointer: fine)';
exports.highDensity = '(-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3 / 2), (min-device-pixel-ratio: 1.5)';
